src/common/dom/apply_themes_on_element.ts

```diff
@@ -1,5 +1,6 @@
+import { Theme } from "../../data/ws-themes";
 import { darkStyles, derivedStyles } from "../../resources/styles";
-import { HomeAssistant, Theme } from "../../types";
+import { HomeAssistant } from "../../types";
```

Comment:
```suggestion
import type { HomeAssistant } from "../../types";
```

Reply:
This type notion is nice to know what are types and what arent. idk if its a performance boost